public/app/core/components/PermissionList/AddPermission.tsx

```diff
@@ -107,31 +106,22 @@ class AddPermissions extends Component<Props, NewDashboardAclItem> {
 
             {newItem.type === AclTarget.User ? (
               <div className="gf-form">
-                <UserPicker
-                  onSelected={this.onUserSelected}
-                  value={newItem.userId.toString()}
-                  className={pickerClassName}
-                />
+                <UserPicker onSelected={this.onUserSelected} className={pickerClassName} />
```

Comment:
New react-select is stateful, so we don't have to manage the value-state ourselves anymore (unless we really need to do that)